files/concourse-worker.sh

```diff
@@ -59,6 +59,6 @@ if [ ! -f "$CONCOURSE/worker_key" ]; then
   { ssh-keygen -y -f ~/.ssh/id_ecdsa "$CONCOURSE/worker_key" >> "$CONCOURSE_KEYS/authorized_worker_keys"; } 2>&- || true
 fi
 
-exec concourse worker --work-dir="$CONCOURSE_WORK" \
+/usr/local/bin/dumb-init /usr/local/bin/concourse worker --work-dir="$CONCOURSE_WORK" \
```

Comment:
Does this actually work when `dumb-init` is not PID 1?  If I'm reading this right, the `sh` that executed this script would end up as PID 1.